src/utils/spec-parser.js

```diff
@@ -177,83 +177,93 @@ function groupByTags(openApiSpec, sortTags = false, sortEndpointsBy) {
 
     methods.forEach((methodName) => {
       let tagObj;
-      let tagText;
       let tagDescr;
 
       if (openApiSpec.paths[path][methodName]) {
         const fullPath = openApiSpec.paths[path][methodName];
+
         // If path.methods are tagged, else generate it from path
-        if (fullPath.tags && fullPath.tags[0]) {
-          tagText = fullPath.tags[0];
-          if (openApiSpec.tags) {
-            tagDescr = openApiSpec.tags.find((v) => (v.name === tagText));
-          }
-        } else {
+        const pathTags = fullPath.tags ? fullPath.tags : [];
+        if (pathTags.length === 0) {
           let firstWordEndIndex = path.indexOf('/', 1);
           if (firstWordEndIndex === -1) {
             firstWordEndIndex = (path.length - 1);
           } else {
             firstWordEndIndex -= 1;
           }
-          tagText = path.substr(1, firstWordEndIndex);
+          pathTags.push(path.substr(1, firstWordEndIndex));
         }
-        tagObj = tags.find((v) => v.name === tagText);
 
-        if (!tagObj) {
-          tagObj = {
-            show: true,
-            name: tagText,
-            description: tagDescr ? tagDescr.description : '',
-            paths: [],
-          };
-          tags.push(tagObj);
-        }
+        pathTags.forEach((tag, index) => {
+          // if multiple tags when can stop or continue to show
+          // path on multiple tag locations
+          if (!allowDuplicatedPathsByTag && index > 0) return;
 
-        // Generate Path summary and Description if it is missing for a method
-        let summary = (fullPath.summary || '').trim() ? fullPath.summary.trim() : (fullPath.description || '-').trim().split('/n')[0];
-        if (summary.length > 100) {
-          summary = summary.split('.')[0];
-        }
-        if (!(fullPath.description || '').trim()) {
-          fullPath.description = ((fullPath.summary || '-').trim());
-        }
+          tag = tag.toLowerCase();
+
+          if (openApiSpec.tags) {
+            tagDescr = openApiSpec.tags.find((v) => (v.name.toLowerCase() === tag));
+          }
 
-        // Merge Common Parameters with This methods parameters
-        let finalParameters = [];
-        if (commonParams) {
-          if (fullPath.parameters) {
-            finalParameters = commonParams.filter((commonParam) => {
-              if (!fullPath.parameters.some((param) => (commonParam.name === param.name && commonParam.in === param.in))) {
-                return commonParam;
-              }
-            }).concat(fullPath.parameters);
+          tagObj = tags.find((v) => v.name === tag);
+          if (!tagObj) {
+            tagObj = {
+              show: true,
+              name: tag,
+              description: tagDescr ? tagDescr.description : '',
+              expanded: tagDescr ? tagDescr.expanded : true,
```

Comment:
I dont get this logic ?

Reply:
Sorry about that row.
I've multiple PR for this project. Expanded is related to the collapsable. Have another branch where all my PR are merged so I can use them. Expanded can be removed from this branch.